IceCreamShop.playground/Contents.swift

```diff
@@ -1 +1,98 @@
+import UIKit
+
+struct Flavor{
+    var name: String
+    var rating: Int
+}
+
+enum Size: Double{
+    case small = 3.99
+    case medium = 4.99
+    case large = 5.99
+}
+
+struct Cone{
+    let flavor: Flavor
+    let topping: String?
+    let size: Size
+    
+    func eat() {
+        print("Mmm! I love \(flavor.name)!")
+    }
+}
+
+class IceCreamShop{
+    var flavors: [Cone]
+    var sizes: [Cone]
+    var toppings: [Cone]
+    var totalSales = 0.00
+    
+    init(totalSales: Double) {
```

Comment:
Your initializer is a little off. Your parameters should take in the variables and then set its value 
 to itself.